docs/supported_ops.md

```diff
@@ -15227,7 +15227,7 @@ are limited.
 <td><b>NS</b></td>
 <td><b>NS</b></td>
 <td> </td>
-<td><b>NS</b></td>
+<td><em>PS<br/>UTC is only supported TZ for child TIMESTAMP;<br/>unsupported child types BINARY, CALENDAR, ARRAY, STRUCT, UDT</em></td>
```

Comment:
This says that it works for window operations too, but I see no tests for window operations.

Reply:
Done, removed the window support.